package.json

```diff
@@ -44,8 +44,8 @@
     "chromatic": "chromatic -t 9q99in2ygnh"
   },
   "peerDependencies": {
-    "react": ">= 16.6.1",
-    "react-dom": ">= 16.6.1",
+    "react": ">= 17.0.1",
+    "react-dom": ">= 17.0.1",
```

Comment:
My one concern with this is that we will be forcing teams using grommet to update react as well. Would it be possible to make the changes on our side that are needed for react 17 in a way that is compatible with both 16 and 17, and then set up our dependencies to allow either?

Reply:
React 17 doesn't have new features from v16 so this should be possible. Would I just need to change the react peerDependencies back to 16.6.1 and leave the react devDependencies at 17.0.1? Or is there more to this

Reply:
I think there's a richer syntax needed, to allow both 16 and 17 major versions